doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,417 @@
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the sedimentation of a single particle in a water tank due to gravity. As other unresolved CFD-DEM applications and examples, we start it but introducing the particle in our domain by running a DEM simulation (using ``lethe-particles``). The DEM simulation generates checkpoint files that will be used by our unresolved CFD-DEM application (``lethe-fluid-particles``) to determine the initial position of the particle.
+
+This is a most probably the simplest application case of CFD-DEM we can think and hence, it is very useful to understand the basics of the method. In Lethe, we use examples such as this to not only guide the user on how to launch a simple example but also track the performance of our code.
+
+In our case, we have used this case to assess the grid convergence of the `quadrature centered method (QCM) <../../../parameters/unresolved-cfd-dem/void-fraction.html>`_, by Geitani and Blais [#geitani2023]_, on reconvering the adequate void fraction and terminal velocity of the sedimenting particle. We use the properties given by Ferreira et al. [#ferreira2023]_ for the particle and fluid.
+
+-------------------
+DEM Parameter File
+-------------------
+
+The syntax of this parameter file is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. All parameter subsections are described in the `parameter section <../../../parameters/parameters.html>`_ of the documentation.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a rectangular-based tank. We use the deal.II GridGenerator in order to generate a hyper rectangle that is subdivided along its height. The following portion of the DEM parameter file shows the function called:
+
+.. code-block:: text
+
+    subsection mesh
+      set type                                = dealii
+      set grid type                           = subdivided_hyper_rectangle
+      set grid arguments                      = 20,40,20:-0.02663,0,-0.02663:0.02663,0.10652,0.02663:true
+      set initial refinement                  = 0
+      set expand particle-wall contact search = false
+    end
+
+.. note::
+    Note that the grid size uses the particle size as reference. This is important because we intended to assess the grid convergence of the case.
+
+.. important::
+    The grid of the particle insertion needs to be the same used in the unresolved CFD-DEM simulation. If you wish to use another mesh refinement, the particle insertion mesh needs to be adapted accordingly.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The time step in this case is the same as the time end. Since we only seek to insert the particle at the top of the channel, we only require 1 insertion time step. We do not need the particle to accelarate, therefore by doing this, the particles will be inserted, but will not fall under the action of gravity.
```

Comment:
```suggestion
The time step in this case is the same as the time end. Since we only seek to insert the particle at the top of the channel, we only require 1 insertion time step. We do not need the particle to accelerate, therefore by doing this, the particle will be inserted, but will not fall under the action of gravity.
```